spec/requests/mutations/editUser_request_spec.rb

```diff
@@ -37,4 +36,35 @@
     expect(edited_user.name).to eq("Edited user name")
     expect(edited_user.email).to eq("one@email.com")
   end
+
+  it "returns an error when the updated email is already associated with another account" do
+    user_1 = User.create!(
+      name: "User 1",
+      email: "user1@email.com"
+    )
+    user_2 = User.create!(
+      name: "User 2",
+      email: "user2@email.com"
+    )
+    error_mutation =
+    <<~GQL
+      mutation createUser {
+          editUser(input:{id: #{user_2.id}, email: "user1@email.com"}) {
+            user {
+              name
+              email
+            }
+          }
+        }
+    GQL
+
+    post '/graphql', params: { query: error_mutation }
+
+    json = JSON.parse(response.body, symbolize_names: true)
+
+    error = json[:errors][0]
+
+
+    expect(error[:message]).to eq("#{user_1.email} is already associated with another account. Your email must be unique.")
+  end
```

Comment:
This is the test for lines 14 + 15 in app/graphql/mutations/edit_user.rb (see comment in file above)